src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Controller.cs

```diff
@@ -175,7 +175,13 @@ private bool StartNewModelComputation(CompletionService completionService, Compl
 
             this.sessionOpt = new Session(this, computation, Presenter.CreateSession(TextView, SubjectBuffer, null));
 
-            sessionOpt.ComputeModel(completionService, trigger, _roles, GetOptions());
+            var options = GetOptions();
+            if (_isDebugger)
+            {
+                options = options.WithChangedOption(EditorCompletionOptions.UseSuggestionMode, true);
```

Comment:
I think we need to take this a step farther - having to toggle the option every time completion comes up isn't what people want I don't think.

Also, what happens if the user hit's Ctrl+Alt+Space to exit suggestion-mode, but the toolbar isn't in suggestion mode?  Did they just toggle their experience in the regular editor?